infrastructure/src/main/java/org/corfudb/infrastructure/BatchProcessor.java

```
@@ -194,7 +194,7 @@ private void processor() {
                                 log.warn("Unknown BatchWriterOperation {}", currOp);
                         }
                     } catch (Exception e) {
-                        log.error("Stream log error. Batch [queue size={}]. StreamLog: [trim mark: {}, tails: {}].",
```

Wont this swallow other exceptions as well?

---

no

---

It looks strange for an exception to use trace level logging.